tests/Feature/QrCodeAdminTest.php

```
@@ -0,0 +1,267 @@
+
+function comercialUser(): User
+{
+    $user = User::factory()->create();
+    $user->assignRole('comercial');
+
+    return $user;
+}
+
+function createQrFixture(array $overrides = []): QrCode
+{
+    $unit = UnitOfMeasure::firstOrCreate(
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\UnitOfMeasure::firstOrCreate().


(staticMethod.notFound)